[IMP] Sign documentation updated (#5558)
odoo/documentation: content/applications/finance/sign.rst

     requested for any field assigned to this role.

.. _sign/itsme:

Itsme®
~~~~~~

Itsme® authentication can be used to allow signatories to provide their identity using itsme®. This
feature is only available in **Belgium** and the **Netherlands**.

The feature can be enabled in :guilabel:`Sign Settings` and applies automatically to the
:guilabel:`Customer (identified with itsme®)` role. To enable it for other roles, go to
:menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra Authentication
Step` column for the role, and select :guilabel:`Via itsme®`.

Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch to any
role configured to use the feature, and click :guilabel:`Validate` and :guilabel:`Send`.

.. image:: sign/itsme-identification.png
   :alt: select customer identified with itsme®

Upon signing the document, the signer completes the :guilabel:`Signature` field and proceeds by
clicking on :guilabel:`Validate & Send Completed Document`, triggering a
:guilabel:`Final verification` page where authentication via itsme® is required.

Signatory hash
==============

> Each time someone signs a document, a hash - a unique ID number - is generated to ensure

Reviewer: ```suggestion
Each time someone signs a document, a hash - a unique digital signature of the operation - is generated to ensure
```

Reviewer: i want to make it somewhat clear that the _content_ the person used for signing (their name, their signature, the input fields they filled in) is used to compute this signature - so any alteration will corrupt the whole 'hash chain' and be visible to the system

here's how chatgpt puts it, which is not a bad start:
> In Odoo Sign, the integrity and inalterability of documents are safeguarded using a hashing technique. At each stage of the signing process, a unique hash is created based on the document's current content and the cumulative hashes of previous steps. This includes any information added by users, like names and signatures. This method ensures that any changes made after a signature is affixed can be easily detected, maintaining the document's authenticity and security throughout its lifecycle."

traceability.

A visual security frame displaying the beginning of the hash is added to the signatures. Internal
users can hide or show it by turning the :guilabel:`Frame` option on or off when signing the
document.

.. image:: sign/sign-hash.png
   :alt: Adding the visual security frame to a signature.

.. _sign/field-types:

Tags
====

Tags can be used to categorize and organize documents, allowing users to search for and filter
documents based on specific criteria quickly.

You can manage tags by going to :menuselection:`Configuration --> Tags`. To create a tag, click
:guilabel:`New`. On the new line, add the :guilabel:`Tag Name` and select a :guilabel:`Color Index`
for your tag.

To apply a tag to a document, use the dropdown list available in your document.

Sign order
==========

When a document needs to be signed by different parties, the signing order lets you control the
order in which your recipients receive it for signature.

> By going to :menuselection:`Configuration --> Settings`, you can :guilabel:`Enable Signing Order`.

Reviewer: no longer needed in 17 btw, signing order is always available as soon as you have more than 1 signer
![image](https://github.com/odoo/documentation/assets/9530767/b678f6f0-a53b-43d5-ac8e-0517191f8bf3)

Reviewer: ![image](https://github.com/odoo/documentation/assets/9530767/4f49aa81-e521-426c-891c-5888db7c957d)

Author: I keep it in mind for V17, thanks!

   :align: center
   :alt: Tip and placeholder example in Odoo Sign

Tags
----

Tags can be used to categorize and organize documents, and allow users to easily search for and
filter documents based on specific criteria.

You can manage tags by going to :menuselection:`Configuration --> Tags`. To create a tag, click
:guilabel:`New`. A line is created. Add the :guilabel:`tag name` and select a
:guilabel:`Color Index` for your tag.

To apply a tag to a document, use the dropdown list available in your document.

Signing order
-------------

By going to :menuselection:`Configuration --> Settings`, you can :guilabel:`Enable Signing Order` to
allow users to define a signing order, where recipients are only notified when it is their turn to
sign.

Expiration and reminders
------------------------

It is possible to set a **validity date** to your document or to send **automatic email reminders**

> to people who still need to sign a document after a set delay.

Reviewer 2: It is possible to set a **validity date** to your document. To obtain the signature on time, you can set up **Automatic email reminders** for people who still need to sign a document.

   :align: center
   :alt: Tip and placeholder example in Odoo Sign

Tags
----

Tags can be used to categorize and organize documents, and allow users to easily search for and
filter documents based on specific criteria.

You can manage tags by going to :menuselection:`Configuration --> Tags`. To create a tag, click

> :guilabel:`New`. A line is created. Add the :guilabel:`tag name` and select a

Author: let's discuss this IRL
